x-pack/plugins/index_management/public/application/components/shared/components/wizard_steps/step_mappings_container.tsx

```diff
@@ -20,13 +21,22 @@ export const StepMappingsContainer: React.FunctionComponent<Props> = ({ esDocsBa
     CommonWizardSteps,
     'mappings'
   >('mappings');
+  const { data: plugins } = useLoadNodesPlugins();
+  const [esNodesPlugins, setEsNodesPlugins] = useState<string[]>([]);
+
+  useEffect(() => {
```

Comment:
Q: is this really necessary? couldn't we just use whatever is returned from `useLoadNodesPlugins`?

```
    <StepMappings
      defaultValue={defaultValue}
      onChange={updateContent}
      indexSettings={getSingleContentData('settings')}
      esDocsBase={esDocsBase}
      esNodesPlugins={plugins ?? []}
    />
```

Reply:
I don't know what I was thinking but yes! 😊